pkg/operator/controller/gatewayapi/crds.go

```diff
@@ -69,6 +79,24 @@ func (r *reconciler) ensureGatewayAPICRDs(ctx context.Context) error {
 	return utilerrors.NewAggregate(errs)
 }
 
+// listUnmanagedGatewayAPICRDs returns a list of unmanaged Gateway API CRDs
+// which exist in the cluster. A Gateway API CRD has "gateway.networking.k8s.io"
+// or "gateway.networking.x-k8s.io" in its "spec.group" field.
+func (r *reconciler) listUnmanagedGatewayAPICRDs(ctx context.Context) ([]string, error) {
+	gatewayAPICRDs := &apiextensionsv1.CustomResourceDefinitionList{}
+	if err := r.cache.List(ctx, gatewayAPICRDs, client.MatchingFields{crdAPIGroupIndexFieldName: gatewayCRDAPIGroupIndexFieldValue}); err != nil {
```

Comment:
Seems like there should be two indexers, one for managed and one for unmanaged.  See https://github.com/openshift/cluster-ingress-operator/pull/1205/files?diff=split&w=1#r2019308004.

Reply:
Same discussion as [this one](https://github.com/openshift/cluster-ingress-operator/pull/1205#discussion_r2021206774).